blackbox/docs/administration/authentication/hba.txt

```diff
@@ -159,3 +159,31 @@ the the ``auth.host_based`` setting, like this:
         config:
           0:
             user: crate
+
+Authenticating to Admin UI
+==========================
+
+.. hide:
+
+    cr> CREATE USER admin;
+    CREATE OK, 1 row affected (... sec)
+
+When trying to access the Admin UI of CrateDB, initially an authentication with
+user ``crate`` will be attempted. If this authentication attempt fails, then
```

Comment:
... with the user defined in the `auth.trust.http_default_user` setting (defaults to `crate`) ...